x/gov/tags/tags.go

```diff
@@ -17,4 +18,5 @@ var (
 	Depositor         = "depositor"
 	Voter             = "voter"
 	ProposalResult    = "proposal-result"
+	ProposalError     = "proposal-error"
```

Comment:
which types of errors are available, and from an ux perspective, why would one want to include them on a tag ?